assets/scss/reset/_forms.scss

```diff
@@ -78,29 +78,6 @@ button,
 	-webkit-appearance: button;
 }
 
-/**
- * Remove the inner border and padding in Firefox.
- */
-
-button::-moz-focus-inner,
-[type="button"]::-moz-focus-inner,
-[type="reset"]::-moz-focus-inner,
-[type="submit"]::-moz-focus-inner {
-	border-style: none;
-	padding: 0;
```

Comment:
I am not sure about this change, but I assume it best to use default styling for the focus elements.